src/iidm/converter/xml/AbstractTransformerXml.hxx

```diff
@@ -219,6 +235,15 @@ void AbstractTransformerXml<Added, Adder>::writeTapChangerStep(const TapChangerS
     writer.writeOptionalAttribute(RHO, tcs.getRho());
 }
 
+template <typename Added, typename Adder>
+void AbstractTransformerXml<Added, Adder>::writeTargetDeadband(double targetDeadband, NetworkXmlWriterContext& context) {
+    IidmXmlUtil::runUntilMaximumVersion(IidmXmlVersion::V1_1(), context.getVersion(), [&context, targetDeadband]() {
+        // in IIDM-XML version 1.0, 0 as targetDeadband is ignored for backwards compatibility
+        // (i.e. ensuring round trips in IIDM-XML version 1.0)
+        context.getWriter().writeOptionalAttribute(TARGET_DEADBAND, targetDeadband, 0.0);
+    });
+}
```

Comment:
Could you complete this method to support v1.2.
https://github.com/powsybl/powsybl-core/pull/958/files#diff-3d00f9a0842fc603305a994c744a6498